src/typeahead/typeahead.js

```diff
@@ -399,13 +399,29 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
     };
   }])
 
-  .filter('typeaheadHighlight', function() {
+  .filter('typeaheadHighlight', ['$sce', '$injector', '$log', function($sce, $injector, $log) {
+
+    var $sanitize;
+    try {
+      $sanitize = $injector.get('$sanitize'); //tries to inject the sanitize service
+    }
+    catch(e) {
+      $log.warn('Unsafe use of typeahead please use ngSanitize'); // Warn the user about the danger
```

Comment:
Shouldn't log here, instead, log below only when the matchItem contains HTML. Users should only see this message if the labels contain HTML tags.